core/src/test/java/org/apache/accumulo/core/iterators/user/FilterTest.java

```diff
@@ -208,9 +208,9 @@ public void test2() throws IOException {
     a = a.deepCopy(null);
     SortedKeyValueIterator<Key,Value> copy = a.deepCopy(null);
     a.seek(new Range(), EMPTY_COL_FAMS, false);
-    assertEquals(size(a), 900);
+    assertEquals(900, size(a));
```

Comment:
Separating out these test fixes into a separate commit (with a separate JIRA) would make me happy as it makes the changeset easier to comprehend if we ever need to go back in history, but I'm not -1 either if you don't.